etna/search/views.py

```diff
@@ -793,20 +751,21 @@ def get_buckets(self) -> Dict[str, Bucket]:
         """
         buckets = {}
         for i, bucket in enumerate(copy.deepcopy(FEATURED_BUCKETS)):
-            response = self.api_result["responses"][i]
-            bucket.result_count = response["hits"]["total"]["value"]
-            bucket.results = response["hits"]["hits"]
+            results = self.api_result[i]
+            bucket.result_count = results.total_count
+            bucket.results = results.hits
             buckets[bucket.key] = bucket
-            self.featured_search_total_count += bucket.result_count
         return buckets
 
-    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
-        data = super().get_datalayer_data(request)
-        total_count = self.featured_search_total_count
-        if total_count > 10000:
-            total_count = 10001
-        data.update(customMetric1=total_count)
-        return data
-
     def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
         return super().get_context_data(buckets=self.get_buckets(), **kwargs)
+
+    def get_result_count(self):
+        """
+        Overrides BaseSearchView.get_result_count() to return the combined
+        totals from all buckets.
+        """
+        total = 0
+        for bucket in self.get_buckets().values():
+            total += bucket.result_count
+        return total
```

Comment:
```suggestion
        for bucket in self.get_buckets().values():
            total += bucket.result_count
        # custom value for customMetric1
        return 10000 if total >= 10000 else total
```
Over here, the actual counts are displayed, but need to set `customMetric1` as per agreed changes mentioned in [DF-334](https://national-archives.atlassian.net/browse/DF-334)